README.md

```diff
@@ -36,7 +36,7 @@ Role Variables
 | ovirt_repositories_ovirt_release_rpm       | UNDEF                 | URL of oVirt release package, which contains required repositories configuration. |
 | ovirt_repositories_use_subscription_manager| False                 | If true it will use repos from subscription manager and the value of <i>ovirt_repositories_ovirt_release_rpm</i> will be ignored. |
 | ovirt_repositories_ovirt_version           | 4.4                   | oVirt release version (Supported versions [4.1, 4.2, 4.3, 4.4]). Will be used to enable the required repositories in case <i>ovirt_repositories_use_subscription_manager</i> is set. |
-| ovirt_repositories_target_host             | engine                | Type of the target machine, which should be one of [engine, host, rhvh]. This parameter takes effect only in case <i>ovirt_repositories_use_subscription_manager</i> is set to True. If incorrect version or target is specified no repositories are enabled. |
+| ovirt_repositories_target_host             | engine                | Type of the target machine, which should be one of [engine, host, rhvh]. In case <i>ovirt_repositories_use_subscription_manager</i> is set to True. If incorrect version or target is specified no repositories are enabled. |
```

Comment:
What's wrong on existing description? For oVirt we have the same repos on both engine and host. For RHV we have different repos for host and engine.

Reply:
I thought to use this variable "ovirt_repositories_target_host"
to distinguish between the engine and to host for "yum module enable".
but maybe it will be better to create a new one, wdyt?

Reply:
Before this patch the role enabled the modules on all machines (engine, host...). I would keep it as it is but update the documentation.

Reply:
for us the issue raised with rhvh maybe it could be problematic too with ovirt node